src/profile/components/ChangePasswordForm.vue

```diff
@@ -38,38 +35,40 @@
       ]"
       :append-icon="showRepeatPassword ? 'mdi-eye' : 'mdi-eye-off'"
       @click:append="showRepeatPassword = !showRepeatPassword"
+      validate-on-blur
     />
 
     <v-btn
-      :disabled="!formValidity || errorMessage === null"
+      type="submit"
       class="submit-btn"
       block
       color="#4f2566"
-      type="submit"
+      :disabled="isButtonDisabled"
       v-on:click="validateForm"
-      ><span
-        >CHANGE PASSWORD
+    >
+      <span v-if="!isButtonDisabled">CHANGE PASSWORD</span>
+      <span v-else>
         <v-progress-circular
           :size="20"
           width="3"
           color="white"
           indeterminate
-          v-if="errorMessage === null"
-        ></v-progress-circular></span
-    ></v-btn>
+        ></v-progress-circular
+      ></span>
+    </v-btn>
   </v-form>
 </template>
 
 <script>
 import validators from "@/shared/form-validators";
 export default {
-  props: ["errorMessage"],
+  props: ["isButtonDisabled"],
```

Comment:
every prop should have specific type
```suggestion
  props: {
       isButtonDisabled: boolean,
  },
```